druid/src/scroll_component.rs

```diff
@@ -124,11 +125,6 @@ impl ScrollbarsState {
 /// [`paint_content`]: struct.ScrollComponent.html#method.paint_content
 #[derive(Debug, Copy, Clone)]
 pub struct ScrollComponent {
-    /// The size of the scrollable content, make sure to keep up this
-    /// accurate to the content being scrolled
-    pub content_size: Size,
-    /// Current offset of the scrolling content
-    pub scroll_offset: Vec2,
     /// Current state of both scrollbars
     pub scrollbars: ScrollbarsState,
```

Comment:
Should ScrollbarsState be collapsed into the parent struct?

Reply:
Yes, that seems to work nicely.